dlib/cmake_utils/find_blas.cmake

```diff
@@ -66,16 +66,29 @@ if (UNIX OR MINGW)
    find_package(PkgConfig)
    pkg_check_modules(BLAS_REFERENCE cblas)
    pkg_check_modules(LAPACK_REFERENCE lapack)
-   if (BLAS_REFERENCE_FOUND AND LAPACK_REFERENCE_FOUND)
+
+   SET(CMAKE_REQUIRED_LIBRARIES "cblas")
```

Comment:
Delete this line. Is this clear why?  You immediately overwrite it below by calling `SET(CMAKE_REQUIRED_LIBRARIES "${BLAS_REFERENCE_LDFLAGS}")` so this line is doing nothing.